queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+SELECT * FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+SELECT * FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
```

Comment:
- [ ] The query requirement here is to list the date of birth of all animals named either "Agumon" or "Pikachu".
 - Selecting from all your database is not the right query. Kindly refactor to capture the `date_of_birth`.
 
 ## Requirement:
![image](https://user-images.githubusercontent.com/50990925/235603935-50ed2470-34a5-48e0-a797-b1f710944f12.png)